File: app/src/main/java/org/ole/planet/myplanet/model/Rows.java
package org.ole.planet.myplanet.model;

public class Rows {
    private String id;

    private String key;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }


    public String getKey() {
        return key;
    }

    public void setKey(String key) {
        this.key = key;
    }

    @Override
    public String toString() {
        return "ClassPojo [id = " + id + ", value =  " + ", key = " + key + "]";
    }
}

File: app/src/main/java/org/ole/planet/myplanet/model/Rows.kt
package org.ole.planet.myplanet.model

class Value {
    var rev: String? = null
}

class Rows {
    var id: String? = null
    var key: String? = null
    var value: Value? = null
    override fun toString(): String {
        return "ClassPojo [id = $id, value = $value , key = $key]"
    }
}
